src/main/java/core/basesyntax/dao/FruitDaoImpl.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.Storage;
+import java.util.Map;
+
+public class FruitDaoImpl implements FruitDao {
```

Comment:
In order to simplify solution and further testing I'd recommend to remove DAO layer from solution and allow handlers to have direct access to storage and actually change quantity, not just calculate it